### ISSUE TITLE
New property for the typisation of data services, using an existing authority table

### ISSUE THREAD

**Issue by JonasPauly-initAG** [SEMICeu/DCAT-AP#405]
**Current Behaviour**

DCAT-AP provides no option for Data Service providers to classify the type of the Data service. 

**Possible Enhancement**

HealthDCAT-AP.de recommends using the Authority Table "Data service type" ([Data service type - EU Vocabularies - Publications Office of the EU](https://op.europa.eu/de/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/data-service-type)) of the EU Publications Office as the range of a new property dcatap:dataServiceType for the class dcat:DataService. This would allow for an improved and more precise description of Data Services.

**Comment by bertvannuffelen** [comment]
In the INSPIRE community this codelist is being used https://inspire.ec.europa.eu/metadata-codelist/SpatialDataServiceType.

In GeoDCAT-AP this is included using http://data.europa.eu/930/serviceType in the https://semiceu.github.io/GeoDCAT-AP/releases/3.0.0/#DataService.servicetype (with suggested codelist https://semiceu.github.io/GeoDCAT-AP/930/releases/3.0.0/#suggestions-of-cv )

**Comment by barthanssens** [comment]
do we really need a new property ? since dcterms:type is already used for typing Dataset, so IMHO it makes sense to just broaden the domain to dcat:Resource (see parapgraph above https://semiceu.github.io/DCAT-AP/releases/3.0.0/#other-controlled-vocabularies) ?

**Comment by matthiaspalmer** [comment]
I agree with @barthanssens we have used dcterms:type for this purpose, pointing to the https://inspire.ec.europa.eu/metadata-codelist/SpatialDataServiceType codelist for some time. I had not noticed the new property http://data.europa.eu/930/serviceType until now.

**Comment by analice1pt** [comment]
I agree with the use of dct:type as it is a very much used property (good also for external interoperability).
